src/extlibs/er_force_sim/src/amun/simulator/simball.h

```diff
@@ -61,7 +69,7 @@ class camun::simulator::SimBall : public QObject
     void sendSSLSimError(const SSLSimError &error, ErrorSource s);
 
    public:
-    void begin();
+    void begin(bool robotCollision);
```

Comment:
please add javadocs to describe robotCollision

Reply:
added